modules/sharedaddy/sharing-sources.php

```diff
@@ -395,6 +431,15 @@ abstract class Deprecated_Sharing_Source extends Sharing_Source {
 		return get_permalink( $post_id );
 	}
 
+	/**
+	 * No AMP display for deprecated sources.
+	 *
+	 * @param \WP_Post $post The current post being viewed.
+	 */
+	final public function get_amp_display( $post ) { // phpcs:ignore
```

Comment:
Here and below, could you add the specific rule we're ignoring, for clarity?

```suggestion
	final public function get_amp_display( $post ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Reply:
Sure! Pushing this change now